src/ESLint.Formatter/test/sarif.js

```diff
@@ -67,6 +67,7 @@ describe("formatter:sarif", () => {
 
             assert.strictEqual(log.runs[0].tool.driver.name, "ESLint");
             assert.strictEqual(log.runs[0].tool.driver.informationUri, "https://eslint.org");
+            assert.strictEqual(log.runs[0].tool.driver.version, undefined);
```

Comment:
>undefined [](start = 64, length = 9)

Kind of a pity that there can't be a test that demonstrates the "real" value. Is there a way to "mock" the source of the version property, so you can see if flows through to the SARIF properly?

Reply:
Thanks. Added new test case, I had to use `rewire` to mock the value, can you please take a look?

Reply:
Nice! Thanks.

---
In reply to: [490200329](https://github.com/microsoft/sarif-sdk/pull/2071#discussion_r490200329) [](ancestors = 490200329)